tests/property/send-to-stata-windows.prop.test.ts

```
@@ -0,0 +1,283 @@
+            (architecture, invalid_checksum) => {
+                const valid_checksum = CHECKSUMS[architecture];
+                
+                // Assume invalid checksum is different from valid one
+                fc.pre(invalid_checksum !== valid_checksum);
+                
+                // Checksum validation should fail for invalid checksums
+                const is_valid = invalid_checksum === valid_checksum;
+                expect(is_valid).toBe(false);
+                
+                // Valid checksums should always pass
+                const valid_is_valid = valid_checksum === valid_checksum;
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)